Kickstarter-iOS/Views/Controllers/ChangePasswordViewController.swift

```diff
@@ -73,23 +74,17 @@ final class ChangePasswordViewController: UIViewController {
       |> UILabel.lens.text %~ { _ in Strings.Confirm_password() }
 
     _ = confirmNewPasswordTextField
-      |> formFieldStyle
-      |> UITextField.lens.secureTextEntry .~ true
```

Comment:
`secureTextEntry` & `placeholder` are duplicates here of `newPasswordFieldAutoFillStyle` who's inheriting it from `passwordFieldStyle`